aiohttp/client_reqrep.py

```diff
@@ -550,6 +556,16 @@ def _headers(self):
     def request_info(self):
         return self._request_info
 
+    @reify
+    def content_disposition(self):
+        raw = self._headers.get(hdrs.CONTENT_DISPOSITION)
+        if raw is None:
+            return None
+        value, params = multipart.parse_content_disposition(raw)
+        params = MappingProxyType(params)
+        filename = multipart.content_disposition_filename(params)
+        return ContentDisposition(value, params, filename)
```

Comment:
Convert params into immutable structure

Reply:
What do you mean? `params` is already converted to MappingProxyType line 565

Reply:
Sorry.
Nevermind